apps/web/app/(authenticated)/(app)/layout.tsx

```diff
@@ -1,15 +1,14 @@
 import { Toaster } from "@/components/ui/toaster";
-import { ReactQueryProvider } from "./ReactQueryProvider";
+import { ReactQueryProvider } from "@/components/dashboard/react-query-provider";
+import { ThemeProvider } from "@/components/dashboard/theme-provider";
```

Comment:
please colocate this file as it's only used once here

Reply:
aight, tho the initial refactor pr was to not colocate components in the app dir. I just think it's more clear to locate things that doesn't affect the router in similar place. But sure since u already done that on the refactor merge will move that way